corehq/apps/smsbillables/models.py

```diff
@@ -427,6 +427,37 @@ def _get_usage_fee(cls, domain, direction):
             )
         return usage_fee
 
+    @classmethod
+    def get_selected_billables(self, datespan):
+        return self.objects.filter(
+            date_sent__gte=datespan.startdate,
+            date_sent__lt=datespan.enddate_adjusted,
+        )
+
+    @classmethod
+    def filter_selected_billables_by_date(self, selected_billables, date_span):
+        return selected_billables.filter(
+            date_created__gte=date_span.startdate,
+            date_created__lt=date_span.enddate_adjusted,
+        )
+
+    @classmethod
+    def filter_selected_billables_show_billables(self, selected_billables, show_billables, ShowBillablesFilter):
+        return selected_billables.filter(
+            is_valid=(show_billables == ShowBillablesFilter.VALID),
+        )
```

Comment:
What is the advantage of passing in `ShowBillablesFilter` vs just importing it?

Reply:
I'm still thinking in C#...